### PR TITLE
sub-#165020167 A user is able to read an article

### PR DESCRIPTION
**Pull request by aaronsekisambu**
### What does this PR do?

- Ensure that a user is able to read an article
- implementing getting an article from the backend

### Questions:

### CODE REVIEW

File: .netlify/state.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+	"siteId": "9a5328dc-4eb1-444d-b54c-f694ca5b5f3f"
```
Comment (ezkemboi) [inline .netlify/state.json:2]: We can add `siteId` to .env file.

**Review by ezkemboi — COMMENTED** [review]
Add `siteId` to .env

**Comment by mendozabree** [comment]
@aaronsekisambu, the PR description is lacking. I am not sure if the rest of the team can review the PR as you are requesting without the necessary instructions and background of the task.

**Comment by ezkemboi** [comment]
@aaronsekisambu, Please read the above comment by @mendozabree.

**Comment by ezkemboi** [comment]
This Pull request should be geared towards PR https://github.com/andela/ah-kgl-avengers-frontend/pull/10

**Pull request closed without merging**